src/Sets/Polygon.jl

```diff
@@ -0,0 +1,38 @@
+export Polygon
```

Comment:
Remember to include this file in LazySets.jl

Reply:
Done